components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -163,7 +163,10 @@ tweaking their $case/namelist_scream.xml file.
     </homme>
 
     <!-- P3 microphysics -->
-    <p3 inherit="physics_proc_base"/>
+    <p3 inherit="physics_proc_base">
+      <do_prescribed_ccn>true</do_prescribed_ccn>
```

Comment:
@hassanbeydoun - this should be false for default runs to continue having the same behavior as current master, right? So using SPA requires do_prescribed_ccn=false, do predict_nc=true, and do_aerosol_rad=true? And no-SPA requires the opposite for all 3 of these variables?

Reply:
ah yes, that's correct. I just wasn't distinguishing between ccn and nc.